airflow/providers/amazon/aws/sensors/glue.py

```diff
@@ -47,14 +47,13 @@ def __init__(self,
         self.aws_conn_id = aws_conn_id
         self.success_states = ['SUCCEEDED']
         self.errored_states = ['FAILED', 'STOPPED', 'TIMEOUT']
+        self.hook = AwsGlueJobHook(aws_conn_id=self.aws_conn_id)
```

Comment:
A hook should not be initialzed during operator/sensor init, because it will be called everytime the webserver parses the dag file the operator is initiated (`task = AwsGlueJobSensor(...)`). The hook initialisation sometimes contains connections to external services which would slow down the Airflow instance.
You only need the hook object in the `poke`/`execute` function when you actually run the task.

Reply:
Got it. I misunderstood how __init__ of sensor works. It's fixed now :)